packages/Frontend/.eslintrc.js

```diff
@@ -30,4 +30,13 @@ module.exports = {
       },
     ],
   },
+  overrides: [
```

Comment:
This was added because I had to mock out firebase for testing. You'll see the mock in another file. This tells our linter that the mock is apart of testing and it lints it slightly differently than it would if it was our regular code.

Reply:
Here's an example of how overrides work within a .eslintrc file. https://eslint.org/docs/user-guide/configuring#example-configuration

Here's where I initially found out how to do this with an example: https://stackoverflow.com/questions/31629389/how-to-use-eslint-with-jest/49211283#49211283